src/__init__.py

```diff
@@ -1,2 +1,4 @@
-from .app_page_scraper import PlayAppPageScraper
-from .betel_errors import BetelError, PlayScrapingError, AccessError
+import src.app_page_scraper
```

Comment:
I think a common pattern is to name the directory that you have called `src` here the project name, so `betel`, even though it looks a bit odd because then you can have the repo called `betel` and then immediately another directory called `betel` inside that, it makes these imports look more sensible. This is more of a personal preference, so if you strongly prefer `src` i think that is also fine.

Reply:
Great suggestion! Thanks :)